run_speakeasy.py

```diff
@@ -46,7 +46,7 @@ def emulate_binary(q, exit_event, fpath, cfg, argv, do_raw, arch='', drop_path='
                 raise Exception('Unsupported architecture: %s' % arch)
 
             sc_addr = se.load_shellcode(fpath, arch)
-            se.run_shellcode(sc_addr)
+            se.run_shellcode(sc_addr, offset=raw_offset or 0)
```

Comment:
I get the following type error here: 
TypeError: emulate_binary() got an unexpected keyword argument 'raw_offset'

Looks like we are missing raw_offset being passed in as an arg to emulate_binary().

Adding the following fixes it:
```suggestion
def emulate_binary(q, exit_event, fpath, cfg, argv, do_raw, arch='', drop_path='', dump_path='',
                                  raw_offset=0):
```